auth/services/selfsigned/test/vc.json

```diff
@@ -0,0 +1,39 @@
+{
+  "@context": [
+    "https://nuts.nl/credentials/v1",
+    "https://www.w3.org/2018/credentials/v1",
+    "https://w3c-ccg.github.io/lds-jws2020/contexts/lds-jws2020-v1.json"
+  ],
+  "id": "did:nuts:8NYzfsndZJHh6GqzKiSBpyERrFxuX64z6tE5raa7nEjm#3ed17821-cedf-4f1b-b29b-393a085b5375",
+  "type": [
+    "NutsEmployeeCredential",
+    "VerifiableCredential"
+  ],
+  "issuer": "did:nuts:8NYzfsndZJHh6GqzKiSBpyERrFxuX64z6tE5raa7nEjm",
+  "issuanceDate": "2023-04-14T10:26:49.975464+02:00",
+  "credentialSubject": [
+    {
+      "@type": "Organization",
+      "id": "did:nuts:8NYzfsndZJHh6GqzKiSBpyERrFxuX64z6tE5raa7nEjm",
+      "member": {
+        "identifier": "user@example.com",
+        "member": {
+          "familyName": "Tester",
+          "initials": "T",
+          "type": "Person"
```

Comment:
shouldn't this be `@type`? If not, `@type` and `type` are the same thing;

"When using a JSON-LD context (see Section [5.3 Extensibility](https://www.w3.org/TR/vc-data-model/#extensibility)), this specification aliases the @type keyword to type to make the JSON-LD documents more easily understood. While application developers and document authors do not need to understand the specifics of the JSON-LD type system, implementers of this specification who want to support interoperable extensibility, do."

(https://www.w3.org/TR/vc-data-model/#types)

Reply:
Just followed the example given on JSON-LD playground. But will change it to `type`